scripts/bootstrap.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Bootstrap virtualenv environment and postgres databases locally.
```

Comment:
The script doesn't seem to do what it says here (nothing to do with postgres in the script?)
When is this intended to be run?